webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/inventory/InventoryFileWatcher.scala

```diff
@@ -169,14 +169,15 @@ class SchedulerMissedNotify(
 ){
 
   /*
-   * List all files that need to be rehad - simply all files older than period
+   * List all files that need to be add again - simply all files older than period
    */
   def listFiles(d: Duration): UIO[List[File]] = {
     IOResult.effect{
       val ageLimit = DateTime.now().minusMillis(d.toMillis.toInt)
-      val filter = (f:File) => (
-           (f.isRegularFile && ageLimit.isAfter(f.lastModifiedTime.toEpochMilli))
-      )
+      val filter = (f:File) => {
+        val ext = f.extension(includeDot = false, includeAll = false).getOrElse("")
+        ( ext == "gz" || ext == "xml" || ext == "ocs" || ext == "sign") && (f.isRegularFile && ageLimit.isAfter(f.lastModifiedTime.toEpochMilli))
```

Comment:
i think we should have a set of authorized extensions, and just have to check if it contains it; I even believe we already have it somewhere.

Reply:
yes, line 102: could you make it a value in the class ?
```val authorizedExtensions = Set("gz", "xml", "ocs", "sign")```

Reply:
not sure where to defined it exactly, perhaps in object `Watchers`